setup.py

```diff
@@ -35,5 +36,5 @@
         'License :: OSI Approved :: Apache Software License',
         'Operating System :: OS Independent',
     ],
-    python_requires='>=3.7',
+    python_requires='>=3.6',
```

Comment:
Why it does not work with 3.6?

Reply:
We can't just use 3.7 since Colab supports only 3.6. We can just use 3.6 but I think covidqa uses 3.7 so it is better to be more general when it doesn't affect us.

Reply:
Oh I see, thanks for the explanation!